src/Frontend/Core/Header/Header.php

```diff
@@ -119,7 +119,12 @@ public function __construct(KernelInterface $kernel)
         $this->meta = new MetaCollection();
 
         // add some default CSS files
-        $this->addCSS('/src/Frontend/Core/Layout/Css/screen.css');
+        $themeCss = $container->getParameter('site.path_www') . '/src/Frontend/Themes/' . Theme::getTheme() . '/Core/Layout/Css/screen.css';
```

Comment:
why is this commit in here?
If I remember it correctly this has been fixed in fork in a different way so you should drop this commit